main.cpp

```diff
@@ -65,10 +62,28 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
+    int soundType = 1;
+    int ingredient = 5;
+    int x = 7;
+
+    float ambient = 3.5f;
+    float surface = 86.4f;
+    float material = 123.2f;
+
+    bool shoots = true;
+    bool jumps = false;
+    bool isGrounded = true;
+
+    double isTurnedOn = 12838.2f;
+    double isTurnedOff = 0.42f;
+    double drinks = 1.3f;
```

Comment:
you're initializing `double`'s with `float`'s.  `double`'s don't have the `f`
- [ ] revise